circle.yml

```diff
@@ -4,4 +4,4 @@ dependencies:
 
 test:
   override:
-    - sbt scripted
+    - sbt ^scripted
```

Comment:
I think `^scripted` should be quoted.
`- sbt '^scripted'`

Reply:
Fixed.